modules/developer-cli-odo-advanced-config.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/configuring-the-odo-cli.adoc
+
+:_content-type: REFERENCE
+[id="developer-cli-odo-advanced-config_{context}"]
+
+= Advanced configuration
+
+You can use the following options for debugging and testing `odo` behavior.
+
+.Environment variables controlling odo behavior
+[cols="1,4",options="header"]
+|===
+|Variable |Usage 
+
+|`PODMAN_CMD`
+|Sets the command that `odo` executes to run the local Podman binary. `podman` by default.
+
+|`DOCKER_CMD`
+|Sets the command that `odo` executes to run the local Docker binary. `docker` by default.
+
+|`ODO_LOG_LEVEL`
+|Sets the log level used by `odo` commands.
+
+|`ODO_DISABLE_TELEMETRY`
+|Disables telemetry collection.
+
+|`GLOBALODOCONFIG`
+|Sets a different location of global preference file `preference.yaml`.
+
+|`ODO_DEBUG_TELEMETRY_FILE`
+|Saves telemetry data to a file rather than sending it to the server.
+
+|`DEVFILE_PROXY`
+|The address for the Devfile registry that integration tests use, rather than `https://registry.stage.devfile.io`
```

Comment:
```suggestion
|The address for the Devfile registry that integration tests use, rather than the one set in list of registries.
```
Or something like that, I can't seem to find proper words.